### ISSUE TITLE
Placable Crafted Items ( Request from Steam )

### ISSUE THREAD

**Issue by synchromesh62** [TheGameCreators/GameGuruRepo#5080]
Placable Crafted Items **Requested by Borg on Steam** 

Again an add-on suggestion. Placable craftable Items that have functions. For example being able to craft work benches and then place them and actually have them wor but have diffent features. Example : Saw table would cut lumber, Firepit would be used for cooking, A crafted foundation could be used to build a house and so on. Again you litterally just got to tweak your current codes to make it possible. I'm not a good coder or I would do it myself, plus your updates would just overwrite it anyway. I do not use Github and will not do so in the future so relay it. I know i'm not the onlyones that want this in. currently your crafting state is very limited and probably preventing most from even using it.

**Comment by superdave69** [comment]
Okay so I game in and joined Github now please. Get on these enhancements. Below is a youtube link of some of what i am working on and will help make sence of what I am looking for. Lee I know your busy and working hard. Thankyou. But please we need these addins.

https://youtu.be/HTUZqRmtsMA

Thankyou.

**Comment by superdave69** [comment]
Alot of those models in the video I made myself like the harvest resources, bananas, logs, log cabin, Iron ore, forge. I use anim8or and Uv mapper to make my models and they go in nice and easy into GGM. Thankyou for the reply and I in no way have been meaning to be a pain. There is a lot of us out here that love what your doing with this software and just eager for more. I'll try to help where i can but I am the worst at coding.

**Comment by Necrym59** [comment]
Already updated the resource node so a specific tool was needed for you and they can respawn, but a crafted placeable object is a bit trickier but i will be looking at somthing like that when i revisit the rpg scripts etc a bit later.

**Comment by superdave69** [comment]
Yes that updated resource node is still not functioning correctely. When you specify a tool ( the correct one ) it states a tool is required. So there is still a bug somewhere. If for example the Signaxe is specified, then it should be able to havested but that's not the case. Below is my game project video. It will show other glitches as well. 


https://youtu.be/vzaF0w_8NnI?si=j0Bt4Iyg07FVfdpG

**Comment by Necrym59** [comment]
It does work correctly, you must put the full name of the custom weapon/tool eg: 'user\axe'   for example you cant just put in 'axe'.

**Comment by superdave69** [comment]
Items when dropped falling through the ground then winding up in the sky,wich I know is a collision issue somewhere just not sure where. And ei will try again as to what you suggest, but I have tried giving the full said name of the Signaxe. Perhaps after the update the bug corrected itself.

**Comment by superdave69** [comment]
Okay I have tried Signaxe clean 2k, Signaxe, Wasteland Weapons Booster Pack/Signaxe. When set up to the extraction node I get that message tool required. The current weapon used is the Signaxe from the Wastland Weapons Booster Pack. What am I getting wrong here?

**Comment by OobieDoob-Benoobie** [comment]
in your first vid, at 3'40
if you turn off "Affected by gravity" and turn on "Is immobile' it should stop the asset from jumping around when you interact with it. Probably just the immobile one would work, but I always do both just to be sure.

**Comment by Necrym59** [comment]
@superdave69 The correct one for the Wasteland 2k Signaxe would be "wasteland\weapons\clean2k\sighnaxe"
it is basically the directory structure -  you can check what it should be by selecting your character start position, select customise and scroll through the weapons list, that will give you the correct naming to put in the behavior.

**Comment by Necrym59** [comment]
The weapons guide walks you through it is as simple as possible, others are doing it just fine just take your time and if you run into a problem ask around others will help you. If you are into doing serious game development of any sort you have to get your hand dirty sometimes unfortunatley.

**Comment by superdave69** [comment]
I'll try that and I get that. But becuase I found it funny I'll post this here:

https://youtu.be/vY7Y2DswHsQ?si=FQKNMJKqarZ-_Q9y

**Comment by superdave69** [comment]
Okay got the signaxe working. Thankyou. I should from here once i figure out how to do Custom weapons be able to get those working as well. Thankyou so much and I am sorry about all the pestering.

**Comment by superdave69** [comment]
Now when you have time the placable Crafted Items would be awsome. By the videos shown you should know what I am going for. Because not only would it work for crafting stations but also structures. Opening up a whole new field of dreams. Placable foundations, walls bridges and of course being able to position the correctly I know a lot of what I am asking is not planned, and if I could code better I would figure it out on my own but coding is my weakest area, and even then am trying to get better. But plzzzzzz plzz plzz pretty plzzz. Thankyou

**Comment by Necrym59** [comment]
Crafted items can have a behavior attached.

**Comment by superdave69** [comment]
Just make sure they're not set up as static meshes.
